src/grammar-registry.js

```diff
@@ -549,10 +549,12 @@ class GrammarRegistry {
   // Extended: Get all the grammars in this registry.
   //
   // Returns a non-empty {Array} of {Grammar} instances.
-  getGrammars (includeTreesitter = false) {
-    let grammars = this.textmateRegistry.getGrammars()
-    if (includeTreesitter) grammars = grammars.concat(Object.values(this.treeSitterGrammarsById))
-    return grammars
+  getGrammars (params = {includeTreeSitter: false}) {
```

Comment:
Turned into options object so we can add / change settings later without breaking changes

Maybe add TM-TS duplicate removal here? It would be slightly easier, and allow it to be used by all packages instead of making them do it themselves